src/main/java/services/ParseSection.java

```diff
@@ -99,4 +107,59 @@ public static SectionAttribute parse(@NotNull String rawData) {
         secData.getOrDefault("Notes", "See Description. None otherwise"),
         secData.get("Room"));
   }
+
+
+  public static void parseBuilding(Map<String, String> secData, String link) {
+    List<String> lines = Utils.asResourceLines("/building.txt");
+    Map<String, String> buildings = new HashMap<>();
```

Comment:
Can you make this a static? It looks like it won't change much in between iterations, so we don't want to recreate it every time.

Reply:
When you said make them static you mean decouple it to a static function or turn it into a static variable? If it's the former then I change it

Reply:
Turn it into a static variable, so that it can be reused between method calls.

Reply:
Yeah, I know you would say that. I changed it to the static variable already in the latest commit

Reply:
Could you change the buildings map to a static? It looks like it's never modified in this block.

Reply:
Otherwise I'm happy with this PR, and I can merge.

Reply:
map is now static